lib/games.rb

```diff
@@ -0,0 +1,31 @@
+require 'csv'
+
+class Games 
+
+    attr_reader :game_id, 
+                :season, 
+                :away_team_id, 
+                :home_team_id, 
+                :away_goals, 
+                :home_goals
+
+    def initialize(game_id,season,away_team_id, home_team_id, away_goals, home_goals)
+      @game_id = game_id
+      @season = season
+      @away_team_id = away_team_id
+      @home_team_id = home_team_id
+      @away_goals = away_goals
+      @home_goals = home_goals
+    end
+
+    def self.load_csv(game_path)
```

Comment:
I was actually thinking of doing it this way, and pass the paths we pass to the stat_tracker into this method. I was going to talk about it after came back together and finished this section and turn this into a separate pull request, but this looks wonderful! I may go back and edit mine real quick to use this same method.